loris/img.py

```diff
@@ -215,21 +215,21 @@ def __getitem__(self, image_request):
 		return fp
 
 	@staticmethod
-	def _link(to,frum):
+	def _link(to, frum):
 		link_dp = path.dirname(frum)
 		if not path.exists(link_dp):
 			makedirs(link_dp)
 		if path.lexists(frum): # shouldn't be the case, but helps debugging
```

Comment:
I think this part might still be problematic, _if_ something goes wrong with calculating a path as I've been seeing in my loris instance.  Or, include my pull request to bail out if somehow and to and from values are the same.